app.js

```diff
@@ -0,0 +1,21 @@
+const taskManager = () => {
+  const inputTodo = document.getElementById("inputTodo");
+  const addTodoButton = document.getElementById("addTodoButton");
+  const listTodo = document.getElementById("listTodo");
```

Comment:
To separate variables containing DOM elements, we can follow the convention of adding `$` sign as a prefix to the variables that contains DOM elements.

```suggestion
  const $todoInput = document.getElementById("inputTodo");
  const $addTodoButton = document.getElementById("addTodoButton");
  const $todoList = document.getElementById("listTodo");
```

```js
const todo = {}
const $todo = document.querySelector('.todo')
```

As you can see, by just looking at the variable name, I can instantly say which one holds the todo DOM element and which on has the todo state